### ISSUE TITLE
Help Wanted: Create animated GIFs for mobile app workflows

### ISSUE THREAD

**Issue by cwarnermm** [mattermost/docs#6657]
## Context

Visuals and short, animated GIFs can help teach visual learners how to perform tasks using the Mattermost mobile app. [Example of a docs page with visuals](https://docs.mattermost.com/collaborate/share-links.html)

## Docs request

- Review all end user product documentation for tasks, features, and outcomes where functionality is missing a visual "show me" element (screenshot or animated GIF). [Example of a docs page with both desktop/web and mobile visual gaps](https://docs.mattermost.com/collaborate/invite-people.html#send-a-direct-invite).
- Capture a short animated GIF or a static image of each end user workflow to add inline on the docs page.

## How you can help
- Use the latest Mattermost mobile app and the standard Mattermost **Denim** theme to capture all workflows.
- Avoid drawing lines, arrows, boxes, or call outs on images.
- Don't forget to include ALT Text descriptions for all new image assets.
- Use existing docs pages with images as a starting point for code guidance. The Docs team can help provide RST syntax support for publication.
- Save static image files in PNG format within the ``/images`` directory in the docs repository.
- Create a docs PR that adds the visuals to the product documentation.

**Comment by Aviraltech** [comment]
Hey @lindy65  , can i work on this issue ?

**Comment by lindy65** [comment]
Yes sure @Aviraltech - I've assigned you

**Comment by cwarnermm** [comment]
@Aviraltech - Are you actively working on this ticket?

**Comment by Rutam21** [comment]
@cwarnermm I believe it would be better if we raise PRs with the new visuals on a per-page basis.
I would like to start with the page that you mentioned in the issue itself.
- https://docs.mattermost.com/collaborate/invite-people.html#

**Comment by cwarnermm** [comment]
@Rutam21 - Excellent approach! I'm very open to splitting out the work needed here into individual issues & PRs. Are you open to creating a new issue specific to the page you'd like to start with?

**Comment by Rutam21** [comment]
Yeah, sure @cwarnermm . I will create a separate issue for that and tag this as the parent issue.

**Comment by cwarnermm** [comment]
Fantastic! Thank you!

**Comment by Rutam21** [comment]
## Checkpoint 1 Reached

- The above list of issues mark the work completion for the [Collaborate within channels](https://docs.mattermost.com/collaborate/collaborate-within-channels.html) section.
- All 16 pages inside it are taken care of and visual workflow guides have been added where necessary.

cc: @cwarnermm

**Comment by Rutam21** [comment]
## Checkpoint 2 Reached

- The above list of issues mark the work completion for the [Communicate with messages and threads](https://docs.mattermost.com/collaborate/communicate-with-messages.html) section.
- All 9 pages inside it are taken care of and visual workflow guides have been added where necessary.

cc: @cwarnermm
